### PR TITLE
chore: bump ethers & typechain

### PR DESCRIPTION
**Pull request by james-a-morris**
Closes ACX-1121

### CODE REVIEW

File: test/SpokePool.Relay.ts [hunk 1]
```diff
@@ -682,6 +682,7 @@ async function testfillRelayWithUpdatedDeposit(depositorAddress: string) {
         updatedMessage,
         consts.modifiedRelayerFeePct, // Applied relayer fee % should be diff from original fee %.
         false,
+        "0",
```
Comment (mrice32) [inline test/SpokePool.Relay.ts:685]: How did this ever work? Do these tests pass?

File: test/SpokePool.SlowRelay.ts [hunk 2]
```diff
@@ -221,6 +221,7 @@ describe("SpokePool Slow Relay Logic", async function () {
           erc20Message,
           0, // Should not have an applied relayerFeePct for slow relay fills.
           true,
+          "9000000000000000000",
```
Comment (mrice32) [inline test/SpokePool.SlowRelay.ts:224]: Same -- how was this working before?

**Review by mrice32 — APPROVED** [review]
LGTM!

File: test/SpokePool.Relay.ts (hunk 1 above)
Comment (james-a-morris) [reply test/SpokePool.Relay.ts:685]: Yep! These tests pass - below are the outputs of the calls to this function commented above.

```
    ✔ Can fill relay with updated fee by including proof of depositor's agreement (38ms)
    ✔ EIP1271 - Can fill relay with updated fee by including proof of depositor's agreement
```

File: test/SpokePool.SlowRelay.ts (hunk 2 above)
Comment (james-a-morris) [reply test/SpokePool.SlowRelay.ts:224]: My understanding is that this was pre ACX 2.5. Here's a passing test from several weeks ago: https://github.com/across-protocol/contracts-v2/actions/runs/4835389290/jobs/8617643141

**Pull request merged**